v4/export/writer.go

```diff
@@ -126,9 +127,12 @@ func writeMetaToFile(ctx context.Context, target, metaSQL string, s storage.Exte
 type CSVWriter struct{ SimpleWriter }
 
 type outputFileNamer struct {
-	Index int
+	Index string
 	DB    string
 	Table string
+
+	id       int
+	idFormat string
```

Comment:
can you make the struct like this?

```go
type outputFileNamer struct {
    Index1 int
    Index2 int
    DB     string
    Table  string
    format string
}
```

use `%09[1]d` to format with first index, `%09[2]d` with second index, and `%09d%04d` for both indices